### PR TITLE
Add reductions to orders

### PR DESCRIPTION
**Pull request by nielsvanoch**

### CODE REVIEW

File: definitions/orders/reduction.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+type: object
+title: Order reduction
+required:
+  - type
+  - amount
```
Comment (bmoelans) [inline definitions/orders/reduction.yaml]: I wouldn't call it amount, take e.g. an action of JIM giving a code you get a topup of 15 for free. So type would be `promo_x` and the second parameter would than be a `value` containing (personalized) code

File: definitions/orders/order.yaml [hunk 2]
```diff
@@ -51,3 +51,8 @@ properties:
     description: A list of payment details
     items:
       $ref: '#/definitions/PaymentDetails'
+  reductions:
+    type: array
```
Comment (bmoelans) [inline definitions/orders/order.yaml]: for sure an array, e.g. and VP and Promo code

File: definitions/orders/create_order.yaml [hunk 3]
```diff
@@ -18,4 +18,10 @@ properties:
     description: ID of the subscription that this order is for
   password:
     type: string
-    description: password required for performing migrations
\ No newline at end of file
+    description: password required for performing migrations
+  reductions:
+    type: array
+    description: >
+      Things you want to use as reductions for the cost of your order.
```
Comment (bmoelans) [inline definitions/orders/create_order.yaml]: reductions and *extra benefits*

**Comment by nielsvanoch** [comment]
@bmoelans @JanJoris I updated this. Could you (re)check?

**Pull request merged**